cardano-cli/ChangeLog.md

```diff
@@ -4,6 +4,23 @@
 
 - Remove cardano-cli address build-script ([PR 4700](https://github.com/input-output-hk/cardano-node/pull/4700))
 
+- New commands for on-chain SPOs polls under `shelley governance`:
+  - `create-poll`:
+      For the current governing entities, as a means to create new polls.
+
+  - `answer-poll`:
+      For participants who want to answer a given poll.
+
+  - `verify-poll`:
+      For anyone who seek to verify a poll entry (e.g. explorers)
+
+  The commands are built to fit and play nicely within the cardano-cli.
+  The poll and answers structures are based on transaction metadata and
+  require to be embedded in an actual transaction. The added commands
+  however only works from metadata and raw "GovernancePoll" envelopes.
+
+  See [CIP proposal](https://github.com/cardano-foundation/CIPs/pull/495) for details.
```

Comment:
The PR seems to be for `CIP-???? | Cardano FUDS (Financial Updates & Disclosure Specification)`

Is that the right PR?

Reply:
Nope, wrong link. It's cardano-foundation/CIPs#496. thanks for catching that, I'll fix the link.

Reply:
https://github.com/input-output-hk/cardano-node/compare/c018de7877eb6c513ddc029224ba2a189c4f6b2c..7adb11180e227089a5aa44845a06737efacf3401